packages/editor/src/components/rich-text/index.native.js

```diff
@@ -49,8 +49,22 @@ export class RichText extends Component {
 	}
 
 	// eslint-disable-next-line no-unused-vars
-	onHTMLContentWithCursor( htmlText, cursorPosition ) {
-		// Descriptive placeholder: This logic still needs to be implemented.
+	onHTMLContentWithCursor( htmlText, start, end ) {
+		if ( ! this.props.onSplit ) {
+			// insert the \n char instead?
```

Comment:
Let's add a ToDo comment here to make sure this is something we come back to at a follow up stage